scripts/process_biomappings.py

```diff
@@ -0,0 +1,48 @@
+# A script to standardize curies in biomappings and invert subject and object, 
+# resulting in MESH to CHEBI mappings for use in the Monarch KG pipeline.
+import sys
+
+import pandas as pd
+import curies
+import sssom
+
+
+converter = curies.Converter([
+    curies.Record(
+        prefix="CHEBI",
+        prefix_synonyms=["chebi"],
+        uri_prefix="http://purl.obolibrary.org/obo/CHEBI_",
+    ),
+    curies.Record(
+        prefix="MESH",
+        prefix_synonyms=["mesh"],
+        uri_prefix="http://purl.obolibrary.org/obo/MESH_",
+    )
+])
+
+try:
+    biomappings_file = sys.argv[1]
+except IndexError:
+    biomappings_file = "https://raw.githubusercontent.com/biopragmatics/biomappings/master/docs/_data/sssom/biomappings.sssom.tsv"
```

Comment:
Isnt there a purl for biomappings? I don't really like this kind of logic here, burying URLs into a script. I would remove the try catch and enforce the parameter being passed in.

Reply:
I don't think there are any purls for biopragmatics mapping files (see https://github.com/biopragmatics/biomappings/ where they offer the repo itself as the source of the files) 

But I'll add in an enforced argument for input file, should be easy enough, likewise with the output name

Reply:
Great.